src/elements/content-sidebar/versions/VersionsSidebarContainer.js

```diff
@@ -166,16 +170,85 @@ class VersionsSidebarContainer extends React.Component<Props, State> {
-            error: shouldShowVersionErrorWithUpsell ? messages.versionNotAvailable : messages.versionFetchError,
-            errorTitle: shouldShowVersionErrorWithUpsell ? messages.versionAccessError : messages.versionServerError,
-            isLoading: false,
-            isWatermarked: false,
-            versionCount: 0,
-            versions: [],
-        });
+        const shouldShowVersions = showUpsell || showUpsellWithPicture;
+
+        if (!shouldShowVersions) {
+            this.setState({
+                error: shouldShowVersionErrorWithUpsell ? messages.versionNotAvailable : messages.versionFetchError,
+                errorTitle: shouldShowVersionErrorWithUpsell
+                    ? messages.versionAccessError
+                    : messages.versionServerError,
+                isLoading: false,
+                isWatermarked: false,
+                versionCount: 0,
+                versions: [],
+            });
+        } else {
+            const isWatermarked = false;
+            const versionLimit = Infinity;
+            const versionCount = 3;
+            const versions = [
+                {
+                    type: 'file_version',
+                    id: '28338883546',
+                    permissions: {
+                        can_delete: false,
+                        can_download: false,
+                        can_preview: false,
+                        can_upload: false,
+                    },
+                    created_at: '2021-11-29T15:01:19-08:00',
+                    modified_at: '2021-11-29T15:01:19-08:00',
+                    modified_by: currentUser,
+                    trashed_at: null,
+                    size: 1875887,
+                    uploader_display_name: 'Jessica Larsen',
+                    version_number: '1',
+                },
+                {
+                    type: 'file_version',
+                    id: '283388835467',
+                    permissions: {
+                        can_delete: false,
+                        can_download: false,
+                        can_preview: false,
+                        can_upload: false,
+                    },
+                    created_at: '2021-11-29T15:01:19-08:00',
+                    modified_by: currentUser,
+                    size: 1875887,
+                    trashed_at: null,
+                    uploader_display_name: 'Jessica Larsen',
+                    version_number: '2',
+                },
+                {
+                    type: 'file_version',
+                    id: '28338883548',
+                    permissions: {
+                        can_delete: false,
+                        can_download: false,
+                        can_preview: false,
+                        can_upload: false,
+                    },
+                    created_at: '2021-11-29T15:01:19-08:00',
+                    currentUser,
+                    size: 1875887,
+                    trashed_at: null,
```

Comment:
a lot of these key-values are identical between the 3 file versions, i'd recommend looking into pulling out the common parts and leveraging the ES6 spread operator to copy them in for each one